logsrvd/logsrvd.h

```diff
@@ -21,11 +21,20 @@
 # error protobuf-c version 1.30 or higher required
 #endif
 
+#include "config.h"
+
+#if defined(HAVE_OPENSSL)
+# include <openssl/ssl.h>
+#endif
+
 #include "logsrv_util.h"
 
 /* Default listen address (port 30344 on all interfaces). */
 #define DEFAULT_LISTEN_ADDR	"*:" DEFAULT_PORT_STR
 
+/* Default timeout value for server socket */
+#define DEFAULT_SOCKET_TIMEOUT_SEC 0
```

Comment:
I think we want a default timeout; 30 seconds seems like a reasonable value.

Reply:
fixed